transit_odp/browse/views/local_authority.py

```diff
@@ -139,77 +140,70 @@ def get_context_data(self, **kwargs):
         context["q"] = self.request.GET.get("q", "")
         context["ordering"] = self.request.GET.get("ordering", "ui_lta_name_trimmed")
         all_ltas_current_page = context["object_list"]
-        ids_list = {}
-
-        for lta in all_ltas_current_page:
-            if lta.ui_lta_name_trimmed not in ids_list:
-                ids_list[lta.ui_lta_name_trimmed] = [lta.id]
-            else:
-                ids_list[lta.ui_lta_name_trimmed].append(lta.id)
-
-        for lta_id_list in ids_list.values():
-            for lta in all_ltas_current_page:
-                if lta.id in lta_id_list:
-                    setattr(lta, "auth_ids", lta_id_list)
-
-                    lta_list = [x for x in all_ltas_current_page if x.id in lta_id_list]
-                    otc_qs = OTCService.objects.get_in_scope_in_season_lta_services(
-                        lta_list
-                    )
-                    if otc_qs:
-                        context["total_in_scope_in_season_services"] = otc_qs
-                    else:
-                        context["total_in_scope_in_season_services"] = 0
-                    requires_attention_data_qs = get_requires_attention_data_lta(
-                        lta_list
-                    )
-                    if requires_attention_data_qs:
-                        context[
-                            "total_services_requiring_attention"
-                        ] = requires_attention_data_qs
-                    else:
-                        context["total_services_requiring_attention"] = 0
-
-                    try:
-                        context["services_require_attention_percentage"] = round(
-                            100
-                            * (
-                                context["total_services_requiring_attention"]
-                                / context["total_in_scope_in_season_services"]
-                            )
-                        )
-                    except ZeroDivisionError:
-                        context["services_require_attention_percentage"] = 0
-                    setattr(
-                        lta,
-                        "services_require_attention_percentage",
-                        context["services_require_attention_percentage"],
-                    )
 
         names = []
         name_set = set()
+        ui_lta_wise_ltas = {}
 
-        all_ltas = self.get_queryset()
+        all_ltas = self.get_model_objects().all()
         for lta in all_ltas:
+            ui_lta_wise_ltas.setdefault(lta.ui_lta_name_trimmed, []).append(lta)
```

Comment:
As discussed - could we verify if there are automated test cases to check the paginated record count when that page has multiple local authorities with the same UI name.